src/Model/AuditReaderInterface.php

```diff
@@ -22,25 +22,33 @@ interface AuditReaderInterface
      * @param string $className
      * @param string $id
      * @param string $revision
+     *
+     * @return object
      */
     public function find($className, $id, $revision);
 
     /**
      * @param string $className
      * @param int    $limit
      * @param int    $offset
+     *
+     * @return array<object>
```

Comment:
We use `object[]` for these cases in other places.

Reply:
Fixed